src/palette/internal/palette/palettecreator.cpp

```diff
@@ -1074,7 +1074,7 @@ PalettePanel* PaletteCreator::newBracketsPalettePanel()
     static Staff bracketItemOwner(gscore);
     bracketItemOwner.setBracketType(static_cast<int>(types.size()) - 1, BracketType::NORMAL);
 
-    for (size_t i = 0; i < types.size(); ++i) {
+    for (int i = 0; i < types.size(); ++i) {
         auto b1 = makeElement<Bracket>(gscore);
         auto bi1 = bracketItemOwner.brackets()[i];
```

Comment:
Here we'd need a cast though, or overload the `QList`´s `[]`operator. Or does it go though without a warning for you?

Reply:
It's problematic that QList etc. use `int` for size and subscripts...
The real solution would be to change the Staff::brackets() to a std::vector instead of a Qt container.
But using a cast would also be okay for now, I think.

Reply:
OK, hold a sec...

Reply:
Another other warnings you're getting?

Reply:
No, now there's nothing about int/size_t etc. any more for me!

(there are some totally different warnings on macOS, but most are not really about the code, there are some strange linker things...)

Reply:
Great!